src/tools/img2vid.py

```diff
@@ -2,35 +2,36 @@
 import sys
 import os
 
+# env var INPUT and OUTPUT_DIR are set in the Dockerfile
+INPUT_DIR = os.environ.get("INPUT_DIR", ".")
+OUTPUT_DIR = os.environ.get("OUTPUT_DIR", ".")
+
 
 def convert(img_dir, output_name="output.mp4", img_prefix="", img_suffix=".jpg", fps=5):
     img_array = []
-    print(f'Frame directory: {img_dir}/')
-    for filename in sorted(os.listdir(img_dir)):
+    img_dir_full_path = os.path.join(INPUT_DIR, img_dir)
+    print(f'Frame directory: {img_dir_full_path}/')
+    for filename in sorted(os.listdir(img_dir_full_path)):
```

Comment:
Maybe just take a list of files and expect them to be listed

Or if string, list and sort files
Else if list, iterate over list.